src/components/VersionSelect/index.tsx

```diff
@@ -21,6 +23,7 @@ export type PublicProps = {
 
 class VersionSelectBase extends React.Component<PublicProps> {
   static defaultProps = {
+    isLoading: false,
```

Comment:
Nit: I wonder if the default should be `true` as it likely will usually be true when this component is initially rendered. I realize that the default doesn't really matter, but if we're going to have one, `true` seems more realistic.

Actually, on second thought, why do we need a default value for `isLoading` at all? It seems reasonable to expect that it will always be provided as a public prop.

Reply:
> Actually, on second thought, why do we need a default value for `isLoading` at all? It seems reasonable to expect that it will always be provided as a public prop.

Oh, that's right. Let's always require this prop.